script/hassfest/brand.py

```diff
@@ -0,0 +1,71 @@
+"""Brand validation."""
+from __future__ import annotations
+
+import voluptuous as vol
+from voluptuous.humanize import humanize_error
+
+from .model import Brand, Config, Integration
+
+BRAND_SCHEMA = vol.Schema(
+    {
+        vol.Required("domain"): str,
+        vol.Required("name"): str,
+        vol.Optional("integrations"): [str],
+        vol.Optional("iot_standards"): [vol.Any("bluetooth", "zigbee", "zwave")],
```

Comment:
Bluetooth is not a generic protocol so that's not a valid option.

`homekit` and `matter` are.
```suggestion
        vol.Optional("iot_standards"): [vol.Any("homekit", "matter", "zigbee", "zwave")],
```

Reply:
homekit added, we should add matter when we have a working matter integration in core.